app/components/SheetMusic/comments/Comment.jsx

```diff
@@ -0,0 +1,52 @@
+
+import React from 'react';
+import FontAwesome from 'react-fontawesome';
+
+let Comment = React.createClass({
+  propTypes: {
+    comment: React.PropTypes.object,
+    level: React.PropTypes.number,
+  },
+
+  getDefaultProps: function() {
+    return {
+      level: 0,
+    };
+  },
+
+  render() {
+    let comment = this.props.comment;
+    let replies = comment.replies;
```

Comment:
Use ES6 destructuring whenever you can for better code readability. For example:

```js
let { comment } = this.props;
let { replies } = comment;
```